heart_animation_2/README.md

```diff
@@ -0,0 +1,27 @@
+# heart_animation_2
+
+A new like button animation.
+
+## Inspiration
+
+This project is inspired from Dribble Design
+
+- [Like Animation](https://dribbble.com/shots/15890672-Like-Animation)
+
+Here is how it looks :
+
+<img src="demo/demo.gif" height=300>
+
+
+## Getting Started
```

Comment:
I think we can remove these below lines just to keep it clean.

Reply:
I made the given changes